src/lib/autocomplete/autocomplete-trigger.ts

```diff
@@ -579,6 +579,10 @@ export class MatAutocompleteTrigger implements ControlValueAccessor, OnDestroy {
     return this._formField ? this._formField.getConnectedOverlayOrigin() : this._element;
   }
 
+  private _getWidth(): number | string {
```

Comment:
Rename this one to `_getPanelWidth` or `_getOverlayWidth` so it's more obvious what it does. Also consider adding a doc string.